system/mp/mpcomm.cpp

```diff
@@ -756,8 +760,7 @@ protected: friend class CMPPacketReader;
                 // loop with short wait time and release CS to allow other side to proceed
                 StringBuffer epStr;
                 unsigned startMs = msTick();
-
-                unsigned loopCnt = ((CONNECT_READ_TIMEOUT / retrycount) / CONNECT_TIMEOUT_INTERVAL) + 1;
+                unsigned loopCnt = CONNECT_TIMEOUT_LOOPCNT;
```

Comment:
More but shorter intervals so another thread gets a chance to aquire a CS was the goal for the smaller loopCnt and higher retrycount.  The error of using a variable retrycount and not a constant was bad.  But I don't think we need to make loopCnt 20x larger to fix that, just make it a constant equal to what is it is the first time through.